packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleTextViewer/SingleTextViewer.spec.js

```diff
@@ -1,23 +1,16 @@
-import { expect } from 'chai'
-import { shallow } from 'enzyme'
 import React from 'react'
+import { render, screen } from '@testing-library/react'
 
 import SingleTextViewer from './SingleTextViewer'
 
-const mockContent = 'test string'
-
-let wrapper
-
 describe('Component > SingleTextViewer', function () {
-  beforeEach(function () {
-    wrapper = shallow(<SingleTextViewer content={mockContent} />)
-  })
-
   it('should render without crashing', function () {
-    expect(wrapper).to.be.ok()
+    render(<SingleTextViewer />)
+    expect(screen).to.be.ok()
```

Comment:
Sorry, dumb question, but won't the `screen` object exported by RTL always be ok?

Reply:
I think for instances where the screen object wouldn't be ok, the code would hit an error before it'd reach the test, like a syntax error, so this test still wouldn't help. I'll delete. 

I ask in the initial comment, I think I could delete all tests in this PR that include `.to.be.ok()`?

Reply:
My thinking was that this test will pass if `screen` contains an empty document. You probably can delete it.

Reply:
You’ve always got the coverage report to refer to if you’re worried that part of the component isn’t tested.